src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -225,7 +226,7 @@
     protected CommonInfo commonInfo = null;
     public final static double NO_LOG10_PERROR = CommonInfo.NO_LOG10_PERROR;
 
-    public final static Set<String> PASSES_FILTERS = Collections.unmodifiableSet(new LinkedHashSet<String>());
+    public final static Set<String> PASSES_FILTERS = Collections.unmodifiableSet(new LinkedHashSet<>());
```

Comment:
correct modifier sort order is
```suggestion
    public static final Set<String> PASSES_FILTERS = Collections.unmodifiableSet(new LinkedHashSet<>());
```
Although I don't see what the point of this is. It's an empty set that can't be modified? If that's what's needed, it already exists as `Collections.emptySet()`.